Source/SwiftLintFramework/Reporters/CodeClimateReporter.swift

```diff
@@ -0,0 +1,65 @@
+#if canImport(CommonCrypto)
+import CommonCrypto
+#else
+import CryptoSwift
+#endif
+import Foundation
+import SourceKittenFramework
+
+#if canImport(CommonCrypto)
+private extension String {
+    func md5() -> String {
```

Comment:
Rather than duplicate this, could you move this to a separate file and make it `internal`?

Reply:
Done. I hope I catch the Swift coding guidelines.